static/js/stream_edit.js

```diff
@@ -537,6 +537,8 @@ export function initialize() {
         const template_data = {
             stream_name: stream.name,
             stream_description: stream.description,
+            max_stream_name_length: page_params.max_stream_name_length,
+            max_stream_description_length: page_params.max_stream_description_length,
```

Comment:
These changes are in the "Manage streams" UI, not anything related to the compose box, and I'd like them to be in a separate commit from compose box changes.